internal/expr/parser.go

```diff
@@ -486,24 +521,76 @@ func (p *Parser) parseIdentifier() (string, bool) {
 	return "", false
 }
 
-// parseColumn parses a column made up of name bytes, optionally dot-prefixed by
+// skipEnclosedParentheses starts from a opening parenthesis '(' and skips until
+// its closing ')', taking into account comments and string literals in between.
+func (p *Parser) skipEnclosedParentheses() (bool, error) {
```

Comment:
This function should be grouped with the other `skip` functions further up the file.